foreman/foreman.spec

```diff
@@ -236,8 +236,8 @@ BuildRequires: npm(diff) >= 3.0.0
 BuildRequires: npm(diff) < 3.1.0
 BuildRequires: npm(expose-loader) < 0.7.0
 BuildRequires: npm(expose-loader) >= 0.6.0
-BuildRequires: npm(extract-text-webpack-plugin) < 3.0.0
-BuildRequires: npm(extract-text-webpack-plugin) >= 2.1.0
+BuildRequires: npm(extract-text-webpack-plugin) < 4.0.0
+BuildRequires: npm(extract-text-webpack-plugin) >= 3.0.0
```

Comment:
I'll update it as it's `"extract-text-webpack-plugin": "^3.0.0"` on package.json